src/main/java/org/reactome/release/qa/check/CoV2EntityCheck.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * This QA check was mostly used during the CoV-1-to-CoV-2 inference process. For now it is being kept, but may
+ * be removed, given it was for a specific curations (August 2020).
+ *
+ * This checks for CoV-1 species/displayNames within all Events/PhysicalEntities contained within the 'SARS-CoV-2 Infection' Pathway.
+ *
+ * @author jcook
+ */
+
+@SliceQACheck
+public class CoV2EntityCheck extends AbstractQACheck {
+
+    @Override
+    public QAReport executeQACheck() throws Exception {
+        QAReport report = new QAReport();
+
+        // Get all contained events within 'SARS-CoV-2 Infection' Pathway, including 'SARS-CoV-2 Infection'.
+        GKInstance cov2InfectionInst = dba.fetchInstance(QACheckerHelper.getCoV2InfectionPathwayDbId());
+        Collection<GKInstance> cov2Events = InstanceUtilities.getContainedEvents(cov2InfectionInst);
+        cov2Events.add(cov2InfectionInst);
+
+        for (GKInstance cov2Event : cov2Events) {
+            Set<GKInstance> cov2Instances = new HashSet<>();
+            cov2Instances.add(cov2Event);
+            // Find all PhysicalEntities within Event
+            if (cov2Event.getSchemClass().isa(ReactomeJavaConstants.ReactionlikeEvent)) {
+                cov2Instances.addAll(InstanceUtilities.getReactionParticipants(cov2Event));
+            }
+            // Iterate through all Events/PhysicalEntities
+            for (GKInstance inst : cov2Instances) {
+                // Get all instances within 'species' and 'relatedSpecies' attributes
+                Set<Long> speciesDbIds = new HashSet<>();
+                if (inst.getSchemClass().isValidAttribute(ReactomeJavaConstants.species)) {
+                    for (GKInstance speciesInst : (Collection<GKInstance>) inst.getAttributeValuesList(ReactomeJavaConstants.species)) {
+                        speciesDbIds.add(speciesInst.getDBID());
+                    }
+                }
+                if (inst.getSchemClass().isValidAttribute(ReactomeJavaConstants.relatedSpecies)) {
+                    for (GKInstance relatedSpeciesInst : (Collection<GKInstance>) inst.getAttributeValuesList(ReactomeJavaConstants.relatedSpecies)) {
+                        speciesDbIds.add(relatedSpeciesInst.getDBID());
+                    }
+                }
+                // Check if any species instance DbIds match the CoV-1 species DbId
+                if (speciesDbIds.contains(QACheckerHelper.getCoV1SpeciesDbId())) {
+                    report.addLine(getReportLine(cov2Event, inst, "CoV-1 species found in CoV-2 instance"));
+                }
+                // Check displayName as well
+                if (inst.getDisplayName().contains("V-1")
+                        || inst.getDisplayName().contains("v-1")
+                        || inst.getDisplayName().contains("v1")
+                        || inst.getDisplayName().contains("V1")) {
+                    report.addLine(getReportLine(cov2Event, inst, "CoV-1 displayName found in CoV-2 instance"));
```

Comment:
So really, this checks that the instances have a name that matches the regex `[vV]-?1`? Could that be put in the description? The description at the top was not very clear.

Reply:
This could also be written as `inst.getDisplayName().matches("^.*?[vv]-?1.*?")` (you may want to test that I got the regex correct!) to match it in a single check or for clarity you can assign the regex in a named variable liked `Cov1Regex`.

Reply:
Your suggested regex works, so I have implemented that